test/ClientWidgetApi-test.ts

```diff
@@ -1437,6 +1437,7 @@ describe("ClientWidgetApi", () => {
     describe("org.matrix.msc2876.read_events action", () => {
         it("reads events from a specific room", async () => {
             const roomId = "!room:example.org";
+            jest.spyOn(clientWidgetApi, "getWidgetVersions").mockReturnValue(new Promise((r) => r([])));
```

Comment:
You could apply the same `mockResolvedValue` shortcut to all the instances of `jest.spyOn(clientWidgetApi, "getWidgetVersions")` here